app/assets/stylesheets/consul.scss

```diff
@@ -35,14 +30,19 @@
       margin-top:4em;
     }
 
-    p{
-      margin: 0 auto;
```

Comment:
Note that after this change, paragraphs have both top and bottom margins. Previously they only had a top margin. Is this change intentional?